test/servers_test.rb

```diff
@@ -41,7 +39,7 @@ class ServersTest < Test::Unit::TestCase
             assert_equal(server_name, created_server.name)
             assert_not_nil(created_server.password)
             assert_instance_of(Array, created_server.users)
-            assert_instance_of(Fixnum, created_server.messages)
+            assert_instance_of(Integer, created_server.messages)
```

Comment:
@luke-hill Thanks for the PR, this is all good to merge except this line, which we've tried before but causes Ruby 2.2 tests to fail, as it expect Fixnum. If you can remove this we can merge the rest.

Reply:
Does this mean the gem has a minimum supported version of 2.2, or lower. This wasn't evident. I'll add this into the gemspec, if you can advise @ what level it needs to be at.